openslides/users/models.py

```diff
@@ -158,11 +159,9 @@ def get_absolute_url(self, link='detail'):
         Returns the URL to the user.
         """
         if link == 'detail':
-            url = reverse('user_detail', args=[str(self.pk)])
+            url = "/users/%s/" % self.pk
         elif link == 'update':
-            url = reverse('user_update', args=[str(self.pk)])
-        elif link == 'delete':
-            url = reverse('user_delete', args=[str(self.pk)])
+            url = "/users/%s/edit/" % self.pk
```

Comment:
But we *have* a url for the update view.

get_absolute_url did never do some actions but only returned an url to a page on which the user could perform an action.

The question is, if we still need the method 'get_absolute_url'. There should be no place in django where an url is needed. The only place where this could be needed is, when  an AjaxView, APIView or RedurectView returns an url to the client. But I don't think this should be necessary. We should allays return the id to a model and create the url with angular